src/modules/forward-back-buttons/forward-back-buttons.component.scss

```diff
@@ -0,0 +1,32 @@
+@import '../../styles/_variables';
+
+/deep/ 
+.stache-forward-back-buttons {
+
+  .stache-nav-forward-back-buttons {
+
+    .stache-nav-list {
+      justify-content: space-between;
+      margin: 10px;
+      flex-direction: row;
+
+      li:first-child {
+        span {
+          &:before {
+            @include import_font_awesome_icon($fa-angle-double-left);
```

Comment:
This is coming from the mixin I made in _font-awesome-variables.scss,  should it be moved into a separate _stache-mixins.scss file?